file_server.rst

```diff
@@ -0,0 +1,64 @@
+.. _file-server-section:
+
+===========
+File server
+===========
+
+The ``File server`` application is built with `Samba
+<http://www.samba.org>`_. It provide shared folders and home directories to
+users and groups of an Active Directory domain.
+
+This application is available by installing a Samba account provider (see
+:ref:`active_directory-section`) with the ``Provide file shares and
+authentication to Windows clients`` switch enabled.
+
+Shared folders
+==============
+
+From the application configuration page, select ``Shared folders`` from
+the left menu and click on :guilabel:`Create shared folder`.
+
+The following parameters are required to create the new shared folder:
+
+- ``Name``: the name of the shared folder
+
+- ``Description``: free text, displayed to SMB clients as the "share comment"
+
+- ``Main group``: select a domain group to assign initial permissions to the share
+
+- ``Initial permissions``: pick one of the three available choices. Note
+  that in any case the ``Domain Admins`` group is initally and recursively
```

Comment:
Fix typo on initially.
Also I'd remove "recursively" because the share is empty and can cause some confusion.
```suggestion
  that in any case the ``Domain Admins`` group is initially 
```